google/cloud/bigquery/quickstart/quickstart.cc

```diff
@@ -0,0 +1,74 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! [START bigquerystorage_quickstart]
+#include "google/cloud/bigquery/bigquery_read_client.gcpcxx.pb.h"
+#include <iostream>
+#include <stdexcept>
+
+namespace {
+void ProcessRowsInAvroFormat(
+    ::google::cloud::bigquery::storage::v1::AvroSchema const&,
+    ::google::cloud::bigquery::storage::v1::AvroRows const&) {
+  // Code to deserialize avro rows should be added here.
+}
+}  // namespace
+
+int main(int argc, char* argv[]) try {
+  if (argc != 3) {
+    std::cerr << "Usage: " << argv[0] << " <project-id> <table>\n";
+    return 1;
+  }
+
+  // project_id should be in the format "projects/<your-gcp-project>"
```

Comment:
For other resources (Bigtable tables, Spanner databases, etc.) we call the fully qualified path `$resource name` (e.g. table name, database name), and the unqualified id, `$resource id` (e.g. table id, database id).

In other words, consider `std::string const project_name = argv[1];` and change the comments accordingly.